microsoft-edge/extensions-chromium/developer-guide/best-practices

```diff
@@ -0,0 +1,27 @@
+---
+title: Best Practices for extensions
+description: Best Practices to implement an extension and list it on the Microsoft Edge Add-ons Store.  
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: extensions
+ms.date: 3/20/2023
+---
+# Best Practices for extensions
+
+Follow these Best Practices to implement your extension and list it on the Microsoft Edge Add-ons Store.
+
+## Design a high-quality extension
+
+Build extensions that conform to Microsoft’s standards for security, privacy, transparency, and user experience.
+
+**Security**: Make sure that your extension does not pose any security threats to users. 
```

Comment:
```suggestion
**Security**: Make sure that your extension doesn't pose any security threats to users. 
```
not sure if i suggested this before and you elected to keep it as two separate words for emphasis.  Corp style is to prefer contractions.  IMO, casual contraction may be sufficient here.